src/chess/board.cc

```diff
@@ -337,8 +337,8 @@ MoveList ChessBoard::GeneratePseudolegalMoves() const {
 }
 
 bool ChessBoard::ApplyMove(Move move) {
-  const auto& from = move.from();
-  const auto& to = move.to();
+  const auto& from = move.GetFrom();
```

Comment:
I think it's better to make `from` and `to` not references now. (I doubt it was a good idea before, either).

Reply:
Done